code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -46,6 +46,8 @@
 /turf/open/Destroy()
 	if(active_hotspot)
 		QDEL_NULL(active_hotspot)
+	if(air)
+		QDEL_NULL(air)
```

Comment:
this seems spooky

Reply:
This value is passed from the old turf to the new one on `/turf/open/ChangeTurf`
The reference should be nulled there